### PR TITLE
[PBA-2837] Live DVR configuration

### PR DESCRIPTION
**Pull request by rcvrgs**
This is related to PR from ios-skin: https://github.com/ooyala/ios-skin/pull/284.

### CODE REVIEW

**Comment by rcvrgs** [comment]
bump, this need to be merged so that ios-skin repo can use it.

**Comment by lental** [comment]
We will need to add this to the schema before we can add this in

**Comment by rcvrgs** [comment]
Ok, I added it to the schema. Please review.

**Comment by lental** [comment]
bump @chibzu @aldigun

File: skin.json [hunk 1]
```diff
@@ -146,6 +146,9 @@
       "height": 81
     }
   },
+  "live": {
+    "dvrEnabled": false
```
Comment (aldigun) [inline skin.json]: Why are we deciding DVR / non-DVR on the skin, rather than having the player to tell the skin whether it is DVR or live content?

Comment (lental) [inline skin.json]: All HLS videos are "DVR", however it depends on how much DVR is enabled.  For example, HLS  live streams could have the bare minimum 10 seconds or 30 minutes.  How does VTC know when DVR is available?

Comment (aldigun) [inline skin.json]: Not with HLS as it is DVR. I remember HDS or HD3 can detect and gives out a flag about DVR is available.

Comment (lental) [inline skin.json]: What if someone wants to disable dvr entirely from the UI? can we have a field for that?

Comment (rcvrgs) [inline skin.json]: Yea, this flag only states if we want the user to see a DVR UI if and only if the stream is DVR capable. If it is disabled/false, then it doesn't matter if the stream comes with DVR, as we won't show any UI for it.

Comment (lental) [inline skin.json]: @aldigun, would you be okay if we renamed the variable to:

forceDvrDisabled

which will allow people to disable DVR if they would like?

**Comment by lental** [comment]
bump @aldigun @chibzu

**Comment by lental** [comment]
@aldigun  @chibzu  can we get another discussion here? this has gone through our sprint a couple of times.  I'd like to get it sorted out

File: skin.json (hunk 1 above)
Comment (chibzu) [inline skin.json]: In the buttons array, we have the "live" indicator entry. Doesn't removing that have the desired affect here? Or is DVR on mobile more complicated than desktop.

Comment (lental) [inline skin.json]: DVR is complicated specifically for HLS - HLS technically always has "DVR" for all live streams (like, minimum 1 chunk of stream). Some customers may want to hide the "DVR" scrubber, and act more like a true live stream.

having a "forceDvrDisabled" is more like "hideScrubberOnLiveStream".  I would be happy to talk about the desired effect here, but I feel like Web will have a similar issue.  I know that HDS or other formats tell you if DVR is part of the stream, but HLS won't

Comment (lental) [inline skin.json]: bump @chibzu @aldigun, still want to get this out. how do you feel about my previous comment?

Comment (chibzu) [inline skin.json]: Got it, fine by me.

Comment (aldigun) [inline skin.json]: this is fine.

**Comment by lental** [comment]
Okay, lovingly renamed this field to be more clear.  Are you okay with this? If so, I'm going to merge it

**Comment by lental** [comment]
bump - Can I merge this?

**Comment by chibzu** [comment]
Will do you one better, merged.

**Comment by lental** [comment]
don't forget to delete the branch!

**Pull request merged**